unit_test/fuzzing/test_requester/test_spdm_requester_get_csr/get_csr.c

```diff
@@ -179,10 +179,10 @@ void libspdm_test_requester_get_csr_case1(void **State)
 {
     libspdm_test_context_t *spdm_test_context;
     libspdm_context_t *spdm_context;
-    uint8_t csr_form_get[LIBSPDM_MAX_CSR_SIZE] = {0};
+    uint8_t csr_form_get[0x1000] = {0};
```

Comment:
You can define `LIBSPDM_MAX_CSR_SIZE` for unit tests / os_stub.

Reply:
Thanks. I have defined the `LIBSPDM_MAX_CSR_SIZE` for unit tests.